src/commands/config/js/stdlib.js

```diff
@@ -0,0 +1,21 @@
+// Stdlib overrides until js_of_ocaml fixes it
```

Comment:
is there an upstream github issue we can reference?

Reply:
https://github.com/ejgallego/jscoq/issues/97

Reply:
> At any rate, we plan to push these changes upstream to JSOO as they are not specific to jsCoq.

Reply:
https://github.com/ocsigen/js_of_ocaml/pull/791

Reply:
It was fixed, but I know that Flow uses old js_of_ocaml